src/NuGet.Clients/NuGet.VisualStudio.OnlineEnvironment.Client/NuGetClientPackage.vsct

```diff
@@ -64,6 +64,20 @@
           <LocCanonicalName>SolutionExplorer.Solution.RestoreNuGetPackages</LocCanonicalName>
         </Strings>
       </Button>
+
+      <Button guid="guidNuGetClientPackageCmdSet" id="cmdIdManageProjectUI" priority="0x0100" type="Button">
+        <Parent guid="guidNuGetClientPackageCmdSet" id="nuGetClientMenuGroup" />
+        <Icon guid="guidToolbarImages" id="bmpAddNuGetPackage" />        
+        <CommandFlag>AllowParams</CommandFlag>
+        <CommandFlag>DefaultInvisible</CommandFlag>
+        <CommandFlag>DynamicVisibility</CommandFlag>
+        <Strings>
+          <CommandName>SolutionExplorer.Project.ManageNuGetPackages</CommandName>
+          <CanonicalName>SolutionExplorer.Project.ManageNuGetPackages</CanonicalName>
+          <LocCanonicalName>SolutionExplorer.Project.ManageNuGetPackages</LocCanonicalName>
+          <ButtonText>Manage &amp;NuGet Packages...</ButtonText>
```

Comment:
Do you need a localized resource here?

Reply:
We don't in our Tools VSCT, but it's an interesting question whether we should.

Reply:
Somehow these do get localized. 
I'm not sure how that happens :/